### ISSUE TITLE
Should imports be unified?

### ISSUE THREAD

**Issue by DennisSmolek** [pmndrs/react-three-jolt#19]
Because the library came from my original folder structure and changed often I left imports mostly directly from folders/directories to help understand where they were coming from. 

However now that this is a standard library for publishing purposes we use barrel exports this could be shortened. 
```js
import { useForwardedRef, useJolt } from '../hooks';
import { getThreeObjectForBody } from '../systems/body-system';
import { vec3 } from '../utils';
import { BodyState } from '../systems';
```
Could be:
```js
import {  BodyState, getThreeObjectForBody, useForwardedRef, useJolt ,vec3, } from 'src';
```

The benefit is brevity, but part of me feels it looses a bit of clarity in where things are. Although most code tools will direct/highlight where modules exist..

**Comment by isaac-mason** [comment]
I think we'll run into issues with circular imports if we do this. I had to remove some similar patterns to fix circular imports in this commit: https://github.com/pmndrs/react-three-jolt/commit/672b08f26a0eae2f2bae66f57ceeac9c03d775ea

**Comment by DennisSmolek** [comment]
> I think we'll run into issues with circular imports if we do this. I had to remove some similar patterns to fix circular imports in this commit: [672b08f](https://github.com/pmndrs/react-three-jolt/commit/672b08f26a0eae2f2bae66f57ceeac9c03d775ea)

Got it, so for now on and when cleaning up I'll use direct imports where possible.
